tests/integration/fr/test_test_case.py

```diff
@@ -344,3 +344,8 @@ def test__edit__reset(test_samples: List[TestCaseRecord]) -> None:
     assert test_case.image_count == 3
     assert test_case.pair_count_genuine == 0
     assert test_case.pair_count_imposter == 3
+
+
+def test__empty_test_case_name() -> None:
+    with pytest.raises(ValueError):
+        TestCase("")
```

Comment:
Please follow the test naming convention — here, this might be `test__init__validate_name`

Reply:
Updating across all tests that were added.